src/screen3d.ts

```diff
@@ -42,21 +42,21 @@ class Screen3D {
    * Convert the alignment property received from location server into format
    * Babylonjs uses.
    *
-   * - location server "alignment" range: -1 to 0
+   * - location server "alignment" range: -1 to 1.0 TODO fix
    * - BabylonJS Vector3 values range from 0 to 1 rad
    */
   sphereRotation(alignment: number): BABYLON.Vector3 {
     return new BABYLON.Vector3(0, Math.abs(alignment) * 2 * Math.PI, 0);
   }
 
-  createSphere(diameter: number, message: MqttMessage): BABYLON.Mesh {
+  createSphere(diameter: number, message: BeaconLocation): BABYLON.Mesh {
     // Create a built-in "sphere" shape; its constructor takes 6 params: name, segment, diameter, scene, updatable, sideOrientation
     const sphere = BABYLON.MeshBuilder.CreateSphere(
       'sphere1',
       {
-        diameterX: diameter + message.xr,
-        diameterY: diameter + message.yr,
-        diameterZ: diameter + message.zr,
+        diameterX: diameter + message.xr * 5,
```

Comment:
By looking at the code alone, I didn't understand where the multiplication by 5 comes from. Maybe a short comment explaining this logic could be warranted.

Reply:
Moved the magic number 5 into a module level variable with this comment:
```
/**
 * The beacon spheres are stretched according to the xr, yr, zr error values
 * received from the server. How significant do we want this effect to be?
 *
 * TODO: find out what would be the correct way to utilize the error values from
 * the location server. AFAIK if we have some confidence interval (like 0.95),
 * we should be able to calculate the exact ellipsoid dimensions, without magic
 * numbers like this one.
 */
```